src/OrbitGl/App.cpp

```diff
@@ -2626,8 +2626,8 @@ orbit_base::Future<std::vector<ErrorMessageOr<void>>> OrbitApp::ReloadModules(
     // (A) deselect functions when the module is not loaded by the process anymore
     if (!process->IsModuleLoadedByProcess(module->file_path())) {
       data_manager_->DeselectFunction(func);
-    } else if (!module->is_loaded()) {
-      // (B) deselect when module does not have functions anymore (!is_loaded())
+    } else if (!module->AreAtLeastFallbackSymbolsLoaded()) {  // TODO: Is this correct?
```

Comment:
Of course these two TODOs are leftovers but this is a delicate part, e.g. @vickyliu-go4it has already brought up something.